pleskdistup/common/src/php.py

```diff
@@ -22,3 +22,11 @@ def get_php_handlers(php_versions: typing.List[version.PHPVersion]) -> typing.Li
 
 def get_outdated_php_handlers(first_modern: version.PHPVersion) -> typing.List[str]:
     return get_php_handlers([php for php in get_known_php_versions() if php < first_modern])
+
+
+def get_php_handlers_by_condition(condition: typing.Callable[[version.PHPVersion], bool]) -> typing.List[str]:
+    return get_php_handlers([php for php in get_known_php_versions() if condition(php)])
+
+
+def get_php_versions_by_condition(condition: typing.Callable[[version.PHPVersion], bool]) -> typing.List[str]:
```

Comment:
If the condition checks `PHPVersion`, then the function should return `List[PHPVersion]`.